meteor-app/imports/ui/ordering/ordering-data-tracker.js

```diff
@@ -2,8 +2,8 @@ import { Meteor } from 'meteor/meteor'
 import { withTracker } from 'meteor/react-meteor-data';
 import { Session } from 'meteor/session'
 
-import OrderingMain from '/imports/ui/ordering/ordering-main'
-import Parts from '/imports/api/parts/parts';
+import OrderingLayout from '/imports/ui/ordering/ordering-layout'
```

Comment:
My intention is to eliminate words that don't add value.
In fact you could just call the file `imports/ui/ordering/layout.js` which will solve the problem. The fewer words you have to read, the easier the code is to understand. Choice of names is really important when coding, to choose short words that still have meaning, and to eliminate redundancies. One of my favourite bad method names was this: `updateCustomerDatebaseRecordSetStatusAvailable()` could probably just have been `setStatus()` or `updateStatus()`

Similarly just call the component `Ordering`